src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -295,6 +297,11 @@ const timeoutBuffer = 400;
  */
 const bidderTimeout = PREBID_TIMEOUT;
 
+/**
+ * useBidCache is false by default then we will set it to true when switch in on and user is in variant
+ */
+const useBidCache = false;
+
```

Comment:
could the `shouldIncludePrebidBidCache` check go here?

Reply:
I am not really sure why we do that from L368-L375 but is there another way to do that? You mean otherwise use `window.guardian.pbjs.useBidCache` instead of assigning them to a variable `pbjsConfig`?

Reply:
Charlotte's suggestion is what I was thinking!

Reply:
@Jakeii I have no idea why I thought about something else or maybe I misread your comment! Thank you @cemms1 for the call to clearly explain that. Done in https://github.com/guardian/commercial/pull/1839/commits/574a3151ae51da6dcfc3d0a13da92d38ee7cb6ba